propolis/src/block/file.rs

```diff
@@ -173,7 +173,7 @@ impl Driver {
 
             let _ = disp
                 .spawn_sync(
-                    format!("file bdev worker {}", i),
+                    format!("file {:?} bdev worker {}", self.fp, i),
```

Comment:
This is no longer necessary after bf46fe953310b93e06df8f1948cb5d5496b6a2cf, but it's probably still good to have distinct names.  The `Debug` output of the `File` might get a little longer for posix_setname, though.  Have you checked to make sure this is fine?

Reply:
FYI, the limit varies per operating system (as does truncation behavior). The limit on helios is 32. Given that's our primary deployment vector you may want to optimize for that rather than the lowest common denominator (but you'll want to verify truncation behavior on other platforms through that interface).